main.go

```diff
@@ -13,18 +14,29 @@ var slavePassword string
 var masterConnection string
 var masterPassword string
 
+var APP_VERSION string = "latest" // Change this by setting compile flag -ldflags "-X main.APP_VERSION=${CI_GIT_TAG}"
+
+func init() {
+	// Metrics have to be registered to be exposed:
+	registerMetrics()
+}
+
 func main() {
 	flag.StringVar(&masterConnection, "master", "redis-master:6379", "The connection string to the Redis master as <hostname/ip>:<port>")
 	flag.StringVar(&slaveConnection, "slave", "redis-slave:6379", "The connection string to the Redis slave as <hostname/ip>:<port>")
 	flag.StringVar(&masterPassword, "master-password", "", "The password used to connect to the master")
 	flag.StringVar(&slavePassword, "slave-password", "", "The password used to connect to the slave")
 	flag.Parse()
 
+	// Iniitialize metrics
+	getHealthStatus()
```

Comment:
Later we could/should invoke this method by a timer that checks for example every 30s (adjustable) the health status otherwise this function/metrics are pretty senseless if you get the health status that is 1 hour old :)